cpp/ps0000_sol.cpp

```diff
@@ -0,0 +1,8 @@
+#include <iostream> // 2/2
+using namespace std;
+int main() // 1/1
+{
+  cout << "Hello, World!\n"; // 4/4
```

Comment:
- [x] used `<<` for outputting instead of `>>`
- [x] used `\n` or `std::endl` in the right way
- [x] avoided syntax error by typing `;` at the end of the line
- [x] output is as required `Hello, World!` in a single line